validation-test/stdlib/FixedPointDiagnostics.swift.gyb

```diff
@@ -2,13 +2,13 @@
 // RUN: %line-directive %t/main.swift -- %target-swift-frontend -parse -verify %t/main.swift
 
 func testUnaryMinusInUnsigned() {
-  var a: UInt8 = -(1) // expected-error {{}} expected-note * {{}}
+  var a: UInt8 = -(1) // expected-error {{}} expected-note * {{}} expected-warning * {{}}
```

Comment:
Yes, that warning is new, I will take a look why that's happening, it's emitted after "error".